compiler/src/iree/compiler/Dialect/LinalgExt/IR/LinalgExtOps.cpp

```diff
@@ -1351,16 +1377,14 @@ SmallVector<AffineMap> AttentionOp::getIndexingMapsForResults() {
 // OnlineAttentionOp
 //===----------------------------------------------------------------------===//
 
-void OnlineAttentionOp::build(::mlir::OpBuilder &odsBuilder,
-                              ::mlir::OperationState &odsState,
-                              ::mlir::TypeRange results, ::mlir::Value query,
-                              ::mlir::Value key, ::mlir::Value value,
-                              ::mlir::Value scale, ::mlir::Value output,
-                              ::mlir::Value max, ::mlir::Value sum,
-                              ::mlir::ArrayAttr indexing_maps,
-                              std::optional<::mlir::Value> mask) {
-  build(odsBuilder, odsState, results, query, key, value,
-        *mask ? *mask : Value(), scale, output, max, sum, indexing_maps);
+void OnlineAttentionOp::build(OpBuilder &odsBuilder, OperationState &odsState,
+                              TypeRange results, Value query, Value key,
+                              Value value, Value scale, Value output, Value max,
+                              Value sum, ArrayAttr indexing_maps,
+                              std::optional<Value> mask) {
+  Value mask_in = mask.value_or(Value());
```

Comment:
mlir uses thisCaseForVariableNames.
```suggestion
  Value maskIn = mask.value_or(Value());
```

Reply:
Makes sense, should this be the same for indexing_maps (or indexingMaps)

Reply:
yes